src/sync/node.js

```diff
@@ -0,0 +1,65 @@
+import PushManagerFactory from './PushManager';
+import { PUSH_DISCONNECT, PUSH_CONNECT } from './constants';
+import FullProducerFactory from '../producer';
+import logFactory from '../utils/logger';
+const log = logFactory('splitio-sync:sync-manager');
```

Comment:
Might be too manija but try ordering the top section by following a criteria and repeating across all files (not just the ones that mirror each other). For example you could do:
Factories
utilities
CONSTANTS
variables/instances

If you are trying to group by semantics, there is either something wrong with the placement (if the constants are shared, don't put them between factories) 
or with the placement of that constants file (as in it's not for the whole sync but only PUSH hence you have them close, move to the push folder maybe so it's easy to extract as a unit, consider re exporting through PUSH manager if that's the case). 
Consider using spaces or comments if grouping by semantic/functionality, but would be better to stay away of the categories might not always be clear.

Apply this comment to all files here and in other PRs, not necessarily right now (but it's harmless and better to get it off the way) but ideally before closing development.

Reply:
thanks. it is good to follow that import order convention